packages/webapp/pages/onboarding.tsx

```diff
@@ -110,7 +123,30 @@ export function OnboardPage(): ReactElement {
   const targetId: string = ExperimentWinner.OnboardingV4;
   const formRef = useRef<HTMLFormElement>();
   const [activeScreen, setActiveScreen] = useState(OnboardingStep.Intro);
-  const enableContentTypeStep = useRef(false);
+  const { updateAdvancedSettings } = useMutateFilters(user);
+  const isSeniorUser =
+    EXPERIENCE_TO_SENIORITY[user?.experienceLevel] === 'senior' ||
+    user?.experienceLevel === 'MORE_THAN_4_YEARS';
+  const { value: seniorContentOnboarding } = useConditionalFeature({
+    feature: feature.seniorContentOnboarding,
+    shouldEvaluate: isSeniorUser,
```

Comment:
We should probably add a state to say only enrol when screen is about to hit.
So default false state, but set to true when onNext passes here:
```
if (seniorContentOnboarding) {
        updateSettingsBasedOnExperience();
      }
```

Reply:
changed to using getFeatureValue inside the onClick rather then useConditionalFeature